### PR TITLE
chore: touch ups to wsproxy UX

### PR DESCRIPTION
**Pull request by Emyrk**
Touchups

# Changes

# Small

- Wsproxy help wording
- Show command help if missing wsproxy edit fields
- remove localhost warning, and remove `*.try.coder` message.

## Wsproxy create text

Screenshot from 2023-07-06 14-29-31
 

## Wsproxy help text in picker

**Styling could be improved**

Screenshot from 2023-07-06 14-42-42

## Wsproxy confirm delete

Screenshot from 2023-07-06 14-45-45

## Workspace proxy table details

**Needs styling**

[Peek 2023-07-06 15-37.webm](https://github.com/coder/coder/assets/5446298/05fcb92c-f9df-46de-9315-16beb32bde6c)

### CODE REVIEW

**Comment by matifali** [comment]
What do you think about allowing adding and deleting workspace proxies from UI?

**Comment by Emyrk** [comment]
> What do you think about allowing adding and deleting workspace proxies from UI?

I am not opposed, but will not do in this PR. This PR is just for minor touchups that came up in a short meeting.

**Comment by Emyrk** [comment]
I think making `create` in the UI is a larger lift given we need an input modal form.

The delete is easier to add to the existing table.

File: site/src/components/Navbar/NavbarView.tsx [hunk 1]
```diff
@@ -269,6 +269,23 @@ const ProxyMenu: FC<{ proxyContextValue: ProxyContextValue }> = ({
...
+            { inlineSize: "200px" },
+            { whiteSpace: "normal" },
+            { textAlign: "center" },
+          ]}
+          onClick={(e) => {
+            // Stop the menu from closing
+            e.stopPropagation()
+          }}
+        >
+          To improve workspace connections, select the closest region.
```
Comment (kylecarbs) [inline site/src/components/Navbar/NavbarView.tsx:286]:
```suggestion
          Reduce workspace latency by selecting the region nearest you.
```
Maybe put in a help tooltip next to that "Only applies to web connections. Local connections will automatically select the nearest region based on latency."

**Review by kylecarbs — APPROVED** [review]

**Pull request merged**